pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -61,15 +66,19 @@ func NewCmdClusterInfo(f cmdutil.Factory, ioStreams genericclioptions.IOStreams)
 	}
 
 	cmd := &cobra.Command{
-		Use:     "cluster-info",
-		Short:   i18n.T("Display cluster info"),
-		Long:    longDescr,
-		Example: clusterinfoExample,
+		Use:                   "cluster-info",
+		DisableFlagsInUseLine: true,
+		Short:                 i18n.T("Display cluster info"),
+		Long:                  longDescr,
+		Example:               clusterinfoExample,
 		Run: func(cmd *cobra.Command, args []string) {
 			cmdutil.CheckErr(o.Complete(f, cmd))
 			cmdutil.CheckErr(o.Run())
 		},
 	}
+
+	cmd.Flags().BoolVar(&o.NoColor, "no-color", false, "Disable colors in the output")
```

Comment:
I'd prefer to use `--color` directly.

Please bind the default value in L65. 

And update as below,

```
cmd.Flags().BoolVar(&o.Color, "color", o.Color, "enable colors in the output")
```

Reply:
Yeah, `--color` is much better, thanks!